pbsmetrics/go_metrics.go

```diff
@@ -218,6 +228,10 @@ func NewMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderName, d
 
 	newMetrics.TimeoutNotificationSuccess = metrics.GetOrRegisterMeter("timeout_notification.ok", registry)
 	newMetrics.TimeoutNotificationFailure = metrics.GetOrRegisterMeter("timeout_notification.failed", registry)
+
+	newMetrics.TCF1Meter = metrics.GetOrRegisterMeter("privacy.tcf.v1", registry)
```

Comment:
I don't think it's clear from the name that this is bid request privacy metrics. We also have user sync related metrics, which I wouldn't mind refactoring in the future. Perhaps "privacy.requests.tcf.v1" or "request_privacy.tcf.v1"?

Reply:
That is a good point, I have not included sync stats.